src/SDKs/DevTestLabs/DevTestLabs.Tests/Tests/BasicTests.cs

```diff
@@ -20,7 +20,7 @@ public void ListLabsTest()
             {
                 var client = GetDevTestLabsClient(context);
 
-                var labs = client.Lab.ListBySubscription().ToList();
+                var labs = client.Labs.ListBySubscription().ToList();
```

Comment:
What are we adding coverage for?  We have two dozen resources which we test in production on a daily basis.  What are these tests for?